config/locales/server.en.yml

```diff
@@ -2604,6 +2605,16 @@ en:
 
         There was an unrecognized error while processing your email and it wasn't posted. You should try again, or [contact a staff member](%{base_url}/about).
 
+    email_reject_attachment:
+      title: "Email Attachment Rejected"
+      subject_template: "[%{email_prefix}] Email issue -- Attachment Rejected"
+      text_body_template:  |
+        We're sorry, but the following attachments in your email message to %{destination} (titled %{former_title}) were rejected: %{rejected_attachments}
+
+        This can be caused by the attached file being too large or by an unauthorised file extension.
```

Comment:
We should at the very least list the authorized extensions. What would be even better would be to tell the reason why an attachment was rejected.

Reply:
there is also a very real  non edge case we should handle here @majakomel which is the default case.

Out of the box we allow no attachments at all except for images. 

I agree with @ZogStriP we should have a bit of a custom message cause this is pretty confusing.

"You may only attach images (png/gif/jpeg) to your post"

"We only allow ".txt, .text" file attachments in emails." 

The very vague message here is a bit confusing.